core/federated/RTI/rti_common.c

```diff
@@ -18,6 +18,20 @@ static rti_common_t* rti_common = NULL;
 // Global variables defined in tag.c:
 extern instant_t start_time;
 
+static tag_t latest_earlier_tag(tag_t tag) {
```

Comment:
It seems like this function could go in tag.h/c, in which case it should be called `lf_tag_latest_earlier` and should be documented something like this:
```suggestion
/**
 * @brief Return the greatest tag earlier than the given tag.
 *
 * If the given tag is `FOREVER_TAG` or `NEVER_TAG`, however, just return the given tag.
 * @param tag The tag.
 */
static tag_t latest_earlier_tag(tag_t tag) {
```

Reply:
I updated the commend and moved it to `tag.h/c` in [2857ab8](https://github.com/lf-lang/reactor-c/pull/349/commits/2857ab87ff374b9a818eedae119327299a9e45fa).